.github/ISSUE_TEMPLATE/content-issue.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Content Issue
+about: Report issues related to the content of the wiki
+title: ''
+labels: content-issue
+assignees: ''
+
+---
+
+**Describe the issue**
```

Comment:
It appears as though your intention was for the issue section headers throughout this pull request to be on a separate line but you would need an empty line between.

Reply:
Oop, thanks for catching that